spec/rubocop/ast/keyword_splat_node_spec.rb

```diff
@@ -39,6 +39,45 @@
     it { expect(kwsplat_node.operator).to eq('**') }
   end
 
+  context 'when forwarded keyword rest arguments', :ruby32 do
+    let(:kwsplat_node) { parse_source(source).ast.children.last.children.last }
+    let(:source) do
+      <<~RUBY
+        def foo(**)
+          { a: 1, ** }
+        end
+      RUBY
+    end
+
+    describe '.new' do
+      it { expect(kwsplat_node).to be_a(described_class) }
+    end
+
+    describe '#hash_rocket?' do
+      it { expect(kwsplat_node).not_to be_hash_rocket }
+    end
+
+    describe '#colon?' do
+      it { expect(kwsplat_node).not_to be_colon }
+    end
+
+    describe '#key' do
+      it { expect(kwsplat_node.key).to eq(kwsplat_node) }
+    end
+
+    describe '#value' do
+      it { expect(kwsplat_node.value).to eq(kwsplat_node) }
+    end
+
+    describe '#operator' do
+      it { expect(kwsplat_node.operator).to eq('**') }
+    end
+
+    describe '#kwsplat_type?' do
+      it { expect(kwsplat_node).to be_kwsplat_type }
+    end
```

Comment:
Great.
We also need `it { expect(kwsplat_node).to be_forwarded_kwrestarg_type }`, right?

Reply:
I see! surely! I've add specs for that.

>  distinguish between `kwsplat` node type and `forwarded_kwrestarg` node type

This worry of mine was unfounded 😅